tdrs-backend/tdpservice/stts/test/test_commands.py

```diff
@@ -0,0 +1,27 @@
+"""Commands tests."""
+
+from django.core.management import call_command
+import pytest
+from ..models import Region, STT
+
+
+@pytest.mark.django_db
+def test_populating_regions_stts():
+    """Test the command for populating regions and STTs."""
+    call_command("populate_stts")
+    assert Region.objects.filter(id=10).exists()
+    assert STT.objects.filter(code="WA", type=STT.EntityType.STATE).exists()
+    assert STT.objects.filter(
+        name="Puerto Rico", type=STT.EntityType.TERRITORY
+    ).exists()
+    assert STT.objects.filter(
+        name="Chickasaw Nation", type=STT.EntityType.TRIBE
+    ).exists()
+
+
+@pytest.mark.django_db
+def test_no_double_population(stts):
```

Comment:
The script was written so as to not duplicate STTs if they already exist in the database. This tests to make sure that is working properly.

Reply:
Does this test rely on `test_populating_regions_stts` to be run first?

Or does passing `stts` as what looks like a method argument trigger running the `stts()` fixture from `conftest.py`?

Reply:
Yes, it triggers running the fixture from `conftest.py`